exec.c

```diff
@@ -336,6 +336,7 @@ RETRACE_REPLACE(execveat)
 int RETRACE_IMPLEMENTATION(fexecve)(int fd, char *const argv[], char *const envp[])
 {
 	int i;
+	rtr_fexecve_t real_fexecve = RETRACE_GET_REAL(fexecve);
```

Comment:
Please don't use functions in declarations. Just split it up:
rtr_fexecve_t real_fexecve;

real_fexecve = RETRACE_GET_REAL(fexecve);